src/sentry/tasks/symbolication.py

```diff
@@ -93,14 +94,18 @@ def should_demote_symbolication(
 # This is f*** joke:
 # The `mock.patch` in `test_symbolication.py` will not work with a static import,
 # so we gotta import the function dynamically here. Great! Hooray!
-def get_native_symbolication_function(data: Any) -> Callable[[Symbolicator, Any], Any] | None:
+def get_native_symbolication_function(
+    data: Any, stacktraces: list[StacktraceInfo]
+) -> Callable[[Symbolicator, Any], Any] | None:
```

Comment:
lol yes, types…
Reminds me of the comment above :-D
Maybe its worth to just get rid of that test? Not sure it provides any value, given that it is mocking this whole implementation.